test/kill_node.sh

```diff
@@ -3,9 +3,15 @@
 # list of process name to be killed
 targetProcess=( "harmony" "bootnode" "soldier" "commander" "profiler" )
 
+NAME="$(uname -s)"
+
 for pname in "${targetProcess[@]}"
 do
-  sudo pkill -9 -e "^(${pname})$"
+  if [ $NAME == "Darwin" ]; then
+    pkill -9 "^(${pname})$"
```

Comment:
so you removed the sudo here, what was the error you had before that ? i assume on your mac your user account has root privilege ? is that the same for all mac OS version ?

Reply:
sudo requires password, but pkill works without it. 

> is that the same for all mac OS version ?
I don't really know, but more likely i have default settings.